### PR TITLE
Wiki: Credit @soopyc

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
@soopyc , looks good?

**Review by networkException — APPROVED** [review]

File: docs/contribute.md [hunk 1]
```diff
@@ -19,3 +19,4 @@ Here is the list and links to donate some of our maintainers.
...
 - [networkException](https://github.com/networkException) - Wiki creator and maintainer. Donate via [GitHub Sponsors](https://github.com/sponsors/networkException).
+- [soopyc](https://github.com/soopyc) - Current maintainer of nixOS. Has helped in reviving its support for T2 Macs. Donate via [GitHub Sponsors](https://github.com/sponsors/soopyc).
```
Comment (networkException) [inline docs/contribute.md:22]: NixOS is the canonical spelling tho

Comment (AdityaGarg8) [reply docs/contribute.md]: Fixed

**Review by soopyc — APPROVED** [review]
LGTM, thanks! <3

**Pull request merged**